LeagueCommissioner/src/main/java/com/revature/beans/Game.java

```diff
@@ -0,0 +1,148 @@
+package com.revature.beans;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+@Entity
+@Table(name = "GAMES")
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE, region = "myAwesomeCache")
+public class Game {
+
+	@Id
+	@Column(name = "GAME_ID")
+	@SequenceGenerator(name = "GAMEID_SEQ", sequenceName = "GAMEID_SEQ")
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "GAMEID_SEQ")
+	private int gameID;
+	
+	@OneToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+	@JoinColumn(name = "TEAM_ID")
+	private int homeTeam;
+	
+	@OneToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+	@JoinColumn(name = "TEAM_ID")
+	private int awayTeam;
+	
+	@Column(name = "GAME_DATE")
+	private String gameDate;
+	
+	@Column(name = "HOME_SCORE")
+	private int homeScore;
+	
+	@Column(name = "AWAY_SCORE")
+	private int awayScore;
+	
+	@Column(name = "IS_OVERTIME")
+	private int isOvertime;

```

Comment:
isOvertime, homeApproved, awayApproved, and commissionerApproved are all 0 or 1, so maybe use the byte data type? Could also make the getters return a boolean.